src/app/clusters/thermostat-server/thermostat-server-events.cpp

```diff
@@ -0,0 +1,153 @@
+/**
+ *
+ *    Copyright (c) 2025 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include "app/clusters/thermostat-server/thermostat-server.h"
+
+#include "app/EventLogging.h"
+
+using namespace chip;
+using namespace chip::app;
+using namespace chip::app::Clusters;
+using namespace chip::app::Clusters::Thermostat;
+using namespace chip::app::Clusters::Thermostat::Attributes;
+
+void EmitSystemModeChangeEvent(EndpointId endpoint, Optional<SystemModeEnum> previousSystemMode, SystemModeEnum currentSystemMode)
```

Comment:
```suggestion
void GenerateSystemModeChangeEvent(EndpointId endpoint, Optional<SystemModeEnum> previousSystemMode, SystemModeEnum currentSystemMode)
```

Same for the others.